frontend/packages/@depmap/data-explorer-2/src/components/ContextBuilderV2/hooks/useNumMatches.ts

```diff
@@ -50,14 +43,15 @@ function useNumMatches(expr: Expr) {
 
       (async () => {
         try {
-          const summary = await fetchContextSummaryV2({
+          const result = await api.evaluateContext({
+            name: "",
```

Comment:
I wasn't sure if this context `name` field was actually being used anymore so I made it optional on the breadbox endpoint. If you're not using it anymore (and we're not supporting legacy contexts in breadbox), maybe I should remove it altogether?

Also, breadbox should be fine with `name` being unset, so you could just exclude it from your request here if it's not doing anything.

Reply:
There are a few operations where the name matters (some plotting stuff that we may or may not wind up implementing in Breadbox) but this certainly isn't one of them. So we could make this endpoint agnostic to it.